doc/tutorial/basics.md

```diff
@@ -48,7 +48,7 @@ Which one you choose depends on the scenario. The normal **Observable** is great
 ## Controlling the flow
 ```js
 // typing "hello world"
-var input = Rx.Observable.fromEvent(document.querySelector('input'), 'keypress');
+var input = Rx.Observable.fromEvent(document.querySelector('input'), 'keyup');
```

Comment:
use `var input = Rx.Observable.fromEvent(document.querySelector('input'), 'input');` instead. (Same for the other line)